tests/cases/compiler/acceptSymbolAsWeakType.ts

```diff
@@ -0,0 +1,22 @@
+// @lib: esnext
+// @target: esnext
+
+const s: symbol = Symbol('s');
+
+const ws = new WeakSet([s]);
+ws.add(s);
+ws.has(s);
+ws.delete(s);
+
+const wm = new WeakMap([[s, false]]);
+wm.set(s, true);
+wm.has(s);
+wm.get(s);
+wm.delete(s);
+
+const wr = new WeakRef(s);
+wr.deref();
+
+const f = new FinalizationRegistry<symbol>(() => {});
+f.registerSymbol(s, null);
+f.unregisterSymbol(s);
```

Comment:
Small typo here causing there to be [errors ](https://github.com/bloomberg/TypeScript/blob/2eb035432c446de9ffee3f43db6da06907ce509b/tests/baselines/reference/acceptSymbolAsWeakType.errors.txt#L25-L28) in the baseline reference for when there shouldn't be errors

```suggestion
f.register(s, null);
f.unregister(s);

```

Reply:
Nice catch, thanks!